examples/websocket-service-basic-auth-ldap-user-store/websocket_service_basic_auth_ldap_user_store.md

```diff
@@ -1,11 +1,14 @@
 # WebSocket service - Basic authentication LDAP user store
 
-A WebSocket service can be secured with Basic authentication and by enforcing authorization optionally. Then, it validates the Basic authentication token sent in the `Authorization` header against the provided configurations. This reads data from the configured LDAP. This stores usernames, passwords for authentication, and scopes for authorization.
+The `websocket:Service` can be secured with basic authentication and optionally, by enforcing authorization. This can be achieved by validating the basic authentication token sent in the initial HTTP upgrade request `Authorization` header with the LDAP server. This server stores the usernames and passwords for the authentication and the scopes for the authorization. An `websocket:Service` can configure the scopes it needs for authorization in the `auth` field of the `@websocket:ServiceConfig` annotation.
```

Comment:
```suggestion
The `websocket:Service` can be secured with basic authentication and optionally, by enforcing authorization. This can be achieved by validating the basic authentication token sent in the initial HTTP upgrade request `Authorization` header with the LDAP server. This server stores the usernames and passwords for the authentication and the scopes for the authorization. Confogure the scopes required by the `websocket:Service`  for authorization in the `auth` field of the `@websocket:ServiceConfig` annotation.
```